src/UpdatesPublisher.js

```diff
@@ -0,0 +1,78 @@
+const {createProducer} = require('windbreaker-service-util/queue')
+const ChangesStream = require('changes-stream')
+const Promise = require('bluebird')
+
+let producerOptions = null
+let amqUrl = null
+let logger = null
+let registryUrl = null
+
+class UpdatesPublisher {
+  configure (_producerOptions, _amqUrl, _registryUrl, _logger) {
+    producerOptions = _producerOptions
+    amqUrl = _amqUrl
+    registryUrl = _registryUrl
+    logger = _logger
+  }
```

Comment:
When you move away from exporting this publisher as a singleton, you can make the `configure` method the class constructor. Just remember to attach the arguments to the object instead of having them floating at the top of the file.